lib/glific/flows/action.ex

```diff
@@ -736,7 +736,7 @@ defmodule Glific.Flows.Action do
 
     results = %{
       "input" => value,
-      "value" => value,
+      "value" => Glific.execute_eex(value),
```

Comment:
Let's run the execute function just after the parse context string. 
Having input value as an EEx does not make sense.

Reply:
value = FlowContext.parse_context_string(context, action.value) |> Glific.execute_eex()

Reply:
made requested changes